python/ray/data/tests/test_batch_mapper.py

```diff
@@ -0,0 +1,133 @@
+import numpy as np
+import pandas as pd
+import pyarrow as pa
+
+import pytest
+from typing import Dict
+from pandas.testing import assert_frame_equal
+
+import ray
+from ray.data.preprocessors import BatchMapper
+
+
+@pytest.fixture
+def test_ds_pandas_format():
```

Comment:
Nit: fixtures shouldn't start with the `test_` prefix, since that's generally reserved for pytest tests.